vulnerable_people_form/form_pages/shared/session.py

```diff
@@ -137,34 +137,14 @@ def get_summary_rows_from_form_answers(excluded_answers=None):
         change_answer_url = append_querystring_params(change_answer_url)
 
         value = {}
-        row = {
-            "key": {
-                "text": question,
-                "classes": "govuk-!-width-two-thirds",
-            },
-            "value": {},
-            "actions": {
-                "items": [
-                    {
-                        "href": change_answer_url,
-                        "text": "Change",
-                        "visuallyHiddenText": question,
-                    }
-                ]
-            },
-        }
         if key == "support_address":
-            value["html"] = "<br>".join(
-                _slice(
-                    [
-                        "building_and_street_line_1",
-                        "building_and_street_line_2",
-                        "town_city",
-                        "postcode",
-                    ],
-                    answer,
-                )
-            )
+            address_lines = [
+                answer.get('building_and_street_line_1'),
```

Comment:
i would use the index syntax here for mandatory fields as it should blow up if first line of address, postcode etc is not in the form answers.

`answer['building_and_street_line_1']`

Reply:
np